srcpkgs/qt6ct/files/qt6ct.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```

Comment:
this script sets/overrides the value set in `/etc/profile.d/qt5ct.sh` (from pkg `qt5ct`). Does that cause issues for qt5 apps?

Reply:
I did some testing and yes it does appear to break qt5 application theming. I have revised the files to now set it to qt5ct as specified on the README of https://github.com/trialuser02/qt6ct. 

I did some testing and it APPEARS that everything works. The qt5 application theming definitely works. However there are still not that many qt6 applications to test so I could only confirm that qt6ct is able to change theming. Also it gives no errors in either qt5ct or qt6ct so I think everything is fine. :thumbsup: